packages/desktop-client/src/components/reports/overview.scss

```diff
@@ -0,0 +1,5 @@
+@use 'react-grid-layout/css/styles.css';
+
+.react-grid-item.react-grid-placeholder {
+  background-color: #8719e0;
```

Comment:
Any way to get this so it is themable?

Reply:
I wish. The lib doesn't expose any prop to easily override the default styles. :( 

We'd need to port the theme-ing solution to use css variables (which would have been better tbh..). But that's quite a large undertaking.